composer.json

```diff
@@ -14,17 +14,22 @@
 		"squizlabs/php_codesniffer": "^2.9.0 || ^3.0.2",
 		"wp-coding-standards/wpcs": "*"
 	},
+	"require-dev" : {
+		"phpcompatibility/php-compatibility": "^8"
```

Comment:
PHPCompatibility will have breaking changes for `9.0.0`.

Reply:
As long as there are no direct references to individual PHPCompatibility sniffs - i.e. inline PHPCS annotations to ignore a particular sniff or references in the ruleset -, the BC-break will be unnoticable.

AFAICS, for this project, some sniffs are excluded in the ruleset, so that will need updating when `9.0.0` comes out, but other than that, you should be fine.